third_party/nvfuser/CMakeLists.txt

```diff
@@ -167,7 +167,7 @@ target_include_directories(${NVFUSER_CODEGEN} PUBLIC
                           "$<BUILD_INTERFACE:${NVFUSER_SRCS_DIR}>"
                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/nvfuser>"
                           )
-set_property(TARGET ${NVFUSER_CODEGEN} PROPERTY CXX_STANDARD 14)
+set_property(TARGET ${NVFUSER_CODEGEN} PROPERTY CXX_STANDARD 17)
```

Comment:
cc @jjsjann123 I bumped this to C++17 to be able to use `std::gcd`. PyTorch is already using C++17, so I believe this is a safe change.